RFC-001-BAM.md

```diff
@@ -168,20 +166,18 @@ The field `type` in a request defines the semantics of the given request. Defini
 
 The design of the `status` field is similar to the status-code in HTTP. It's goal is to be easily machine-readable and assign semantics to status codes that are related to each other. See [[1], Section 3.3]( https://tools.ietf.org/html/rfc3117#section-3.3) for more information on the design of status codes.
 
-##### Application data
-
-This transport protocol defines generic functionality for our application protocol `GANP`. In order to avoid conventions like prefixing custom headers with `X-` as in HTTP, the `REQUEST` and the `RESPONSE` frame reserves a dedicated place for sending application data to other nodes. 
-
 ##### Headers
 
-`Headers` and `Body` are supposed to be used by the application protocol (for the beginning, just GANP). Similar to HTTP, application protocols MAY include some kind of 'Content-Type' in the protocol headers in order to describe the encoding of the payload.
+`Headers` and `Body` are supposed to be used by an application protocol defined on top of `BAM`.
+Application protocols can be defined by defining `REQUEST` types and with them, the semantics of certain headers and the body of a `REQUEST`.
```

Comment:
:upside_down_face: To avoid repetition.
```suggestion
Application protocols can be described by defining `REQUEST` types and with them, the semantics of certain headers and the body of a `REQUEST`.
```